### ISSUE TITLE
Intermittent/recurring issue - Seeing increased numbers of 503 errors on Jenkins pod starts this week

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#3934]
This problem has been seen in the past and seems to have become very common this week - after a user env reset, the Jenkins pod encounters multiple 503 errors on starting. The result is that it can require several minutes for the Jenkins pod to start. 

I have seen this on all clusters multiple times this week - example:

```
10:26:33 AM | jenkins-1-6z758 | Pod | Warning | Unhealthy | Readiness probe failed: HTTP probe failed with statuscode: 503 12 times in the last minute
10:24:32 AM | jenkins-1-6z758 | Pod | Warning | Unhealthy | Readiness probe failed: Get http://10.130.111.25:8080/login: dial tcp 10.130.111.25:8080: getsockopt: connection refused2 times in the last 2 minutes
```

**Comment by ldimaggi** [comment]
This is still a problem today:

```

9:38:56 AM	jenkins-1-8cp8x	Pod	Warning	Unhealthy 	Readiness probe failed: HTTP probe failed with statuscode: 503
12 times in the last 2 minutes
9:36:56 AM	jenkins-1-8cp8x	Pod	Warning	Unhealthy 	Readiness probe failed: Get http://10.130.89.147:8080/login: dial tcp 10.130.89.147:8080: getsockopt: connection refused
2 times in the last 2 minutes

```

**Comment by ldimaggi** [comment]
Still consistently seeing this - the user account is provisioned on the starter-us-east-2 cluster:
![screenshot from 2018-08-02 14-43-28](https://user-images.githubusercontent.com/642621/43604048-bd7850c8-9662-11e8-8583-3b46f2470c1c.png)

The problem eventually resolves itself, but it can take up to 15 minutes to resolve.

**Comment by ldimaggi** [comment]
Setting the priority to P2 as, while this is not a blocker for the user, it causes frequent/constant delays.

**Comment by piyush1594** [comment]
@ldimaggi The issue has been identified and resolved today as a part of #3956 It is on way to prod. @aslakknutsen Can we get a tenant updated to version 4.0.97. PR already raised at https://github.com/openshiftio/saas-openshiftio/pull/1014

**Comment by ldimaggi** [comment]
Still seeing this problem:

![screenshot from 2018-08-02 17-09-38](https://user-images.githubusercontent.com/642621/43611384-f1192a4c-9676-11e8-9172-d28eb5afe483.png)

**Comment by hrishin** [comment]
@ldimaggi readiness probs could show such behavior for some time since Jenkins is still taking some time to start. Now the fix we have production has reduced Jenkins access time significantly. 
As a user are you facing too much delays to access Jenkins dashboard or start builds?

**Comment by ldimaggi** [comment]
The problem described in this issue is now (mostly) being masked by this issue: https://github.com/openshiftio/openshift.io/issues/4121

**Comment by piyush1594** [comment]
@ldimaggi This issue is not masked by #4121 That is totally a different issue. More details can be seen here https://github.com/openshiftio/openshift.io/issues/4121#issuecomment-410301108

**Comment by hrishin** [comment]
@ldimaggi still observing this issue on prod?

![image](https://user-images.githubusercontent.com/28615674/43781114-cb7554da-9a79-11e8-806f-154621f691e0.png)

503 frequency has dropped extensively by now. Other improvements are on the way which could descrese Jenkins boot time from 2-3 minutes to around 40 seconds .

**Comment by piyush1594** [comment]
Basically, this is not an issue but these warnings are events logs for the pod. We have fixed this warning for idling/unidling condition but it will still show for reset environment thing and just add the logs, so no problem or effects. 

It is available in Jenkins Version 4.0.106 in prod. Thanks, closing this.
